libbeat/docs/release-notes/highlights/highlights-7.7.0.asciidoc

```diff
@@ -18,9 +18,131 @@ For a list of bug fixes and other changes, see the {beats}
 
 // tag::notable-highlights[]
 
-//[float]
-//==== highlight
+[float]
```

Comment:
Noting here that the release blog link for observability needs to be updated because it points to 7.6. I'll update this when 7.7 goes live.